tasks/check_html_fixtures.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: false
+require 'diff/lcs'
+require 'diff/lcs/hunk'
+require 'fileutils'
+require "htmlbeautifier"
+
+
+# Get top-level Ruby examples
+file_paths = Dir["examples/*.rb"]
```

Comment:
I think you're lucky you picked this exact set of examples. Anything with a local asset server is going to cause you trouble -- its port number is going to change run-to-run. Worse yet, we *want* its port number to change run-to-run. Though we can probably make it work somehow.

Reply:
Yeah, good to know. Happy to fix forward ways to make it increasingly resilient. Though it's an extra headwind being a new ci step I still want to keep it friendly as possible and there are multiple things we can look at too going forward :)